src/pages/Home/Home.styles.js

```diff
@@ -0,0 +1,53 @@
+import Styled from 'styled-components';
+
+
+const ButtonWrapper = Styled.div`
+  width: 217px;
+  height: 36px;
+`;
+
+const StyledA = Styled.a`
+  letter-spacing: -.1px
+  margin-left: -.5px;
+  margin-right: -5px;
+`;
+
+const StyledA2 = Styled.a`
+  letter-spacing: .06px;
+  margin-left: 4px;
```

Comment:
You only need this margin because there is no space before this link

Reply:
Btw as an alternative of `&nbsp;` you can also use {" "} at the end of line or a new line